IntelliTect.TestTools.Selenate/IntelliTect.TestTools.Selenate/WebElementExtensions.cs

```diff
@@ -48,15 +48,44 @@ public static void ScrollIntoView(this IWebElement element, IWebDriver driver, i
             js.ExecuteScript($"window.scrollTo(0,{position})");
         }
 
+        /// <summary>
+        /// Clears the current text, 
+        /// uses SendKeys to send the specified value to the element, 
+        /// </summary>
+        /// <param name="value">Value to send to the element</param>
+        public static void ClearAndSendKeys(this IWebElement element, string value)
+        {
+            element.Clear();
+            element.SendKeys(value);
+            if (element.GetAttribute("value") != value)
+            {
+                element.Clear();
+                element.SendKeys(value);
+            }
+        }
+
+        /// <summary>
+        /// Clears the current text, 
+        /// uses SendKeys to send the specified value to the element, 
+        /// then tabs out of the field
+        /// </summary>
+        /// <param name="value">Value to send to the element</param>
+        public static void ClearAndSendKeysAndTab(this IWebElement element, string value)
+        {
+            ClearAndSendKeys(element, value);
+            element.SendKeys(Keys.Tab);
+        }
+
         /// <summary>
         /// Waits for the element to be in a valid state, then clears the current text, 
         /// uses SendKeys to send the specified value to the element, 
         /// then tabs out of the field or throws after a certain amount of time
         /// </summary>
         /// <param name="value">Value to send to the element</param>
+        //[Obsolete("Use ElementHandler.SendKeysWhenReady for a common implementation of a SendKeys wrapped by a WebDriverWait implementation")]
```

Comment:
Commented out obsolete attribute? Did you mean to comment it out? Same comment on the methods below.

Reply:
Yeah... I probably jumped the gun too quickly on obsoleting some things, and I think I've obsoleted something every version for the last three merges. So I commented it out and am intending to push out a beta package to use at a couple of our clients (which, actually, brings up the super important point of: should beta packages ever come out of master? As soon as I typed this response it dawned on me that they probably shouldn't.)